tests/syslog/test_syslog.py

```diff
@@ -16,26 +16,25 @@
 SYSLOG_MESSAGE_TEST_TIMEOUT = 10
 
 
-@pytest.fixture(scope="module")
-def config_syslog_srv(ptfhost, duthost):
+def config_syslog_srv(ptfhost):
     logger.info("Configuring the syslog server")
 
     # Add the imudp configuration if not present
-    ptfhost.shell("sed -ni '/module/!p;$a module(load=\"imudp\")' /etc/rsyslog.conf")
-    ptfhost.shell("sed -i '/input(type/!p;$a input(type=\"imudp\" port=\"514\")' /etc/rsyslog.conf")
-
-    # Remove local /var/log/syslog
-    ptfhost.shell("rm -rf /var/log/syslog")
+    ptfhost.shell("sed -i -e '/^input(type/d' -e '/^module/d' /etc/rsyslog.conf; sed -i -e '$amodule(load=\"imudp\")' -e '$ainput(type=\"imudp\" port=\"514\")' /etc/rsyslog.conf")
 
-    # Restart Syslog Daemon
-    ptfhost.shell("service rsyslog restart")
+    # Stop Syslog Daemon
+    logger.info("Restarting rsyslog service")
```

Comment:
Should the comment still be "Restart" since we are still stopping and eventually restarting this service in the next part?

Reply:
Oh, this one was because I consolidated the 3 command to one. I'll fix this comment

Reply:
Thanks! Fixed.